app/code/Magento/ProductAlert/Controller/Unsubscribe/Stock.php

```diff
@@ -21,16 +23,33 @@ class Stock extends UnsubscribeController
     protected $productRepository;
 
     /**
-     * @param \Magento\Framework\App\Action\Context $context
-     * @param \Magento\Customer\Model\Session $customerSession
-     * @param \Magento\Catalog\Api\ProductRepositoryInterface $productRepository
+     * @var Magento\Store\Model\StoreManagerInterface
+     */
+    protected $storeManager;
+
+    /**
+     * @var \Magento\ProductAlert\Model\StockFactory
+     */
+    protected $stockFactory;
```

Comment:
Please use `private` access modifier according to 2.7 of [Technical Guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html).

Reply:
@swnsma
done